tests/Database/DatabaseEloquentFactoryTest.php

```diff
@@ -242,16 +238,11 @@ public function test_morph_to_relationship()
 
     public function test_belongs_to_many_relationship()
     {
-        $users = FactoryTestUserFactory::times(3)
-                        ->hasAttached(
-                            FactoryTestRoleFactory::times(3)->afterCreating(function ($role, $user) {
-                                $_SERVER['__test.role.creating-role'] = $role;
-                                $_SERVER['__test.role.creating-user'] = $user;
-                            }),
-                            ['admin' => 'Y'],
-                            'roles'
-                        )
-                        ->create();
+        $roleFactory = FactoryTestRoleFactory::times(3)->afterCreating(function ($role, $user) {
+            $_SERVER['__test.role.creating-role'] = $role;
+            $_SERVER['__test.role.creating-user'] = $user;
+        });
+        FactoryTestUserFactory::times(3)->hasAttached($roleFactory, ['admin' => 'Y'], 'roles')->create();
```

Comment:
Revert the code style changes you made here and only focus on the PHPUnit fixes.